File: app/src/main/java/edu/ucdenver/fallnotification/Patient.java
package edu.ucdenver.fallnotification;

import java.util.Calendar;

public class Patient {
    private String firstName;
    private String lastName;
    private int roomNumber;
    private int age;
    public Calendar[] sessions;

    private Patient(String firstName, String lastName, int roomNumber, int age) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.roomNumber = roomNumber;
        this.age = age;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public int getRoomNumber() {
        return roomNumber;
    }

    public void setRoomNumber(int roomNumber) {
        this.roomNumber = roomNumber;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
}

File: app/src/main/java/edu/ucdenver/falldetection/Patient.kt
package edu.ucdenver.falldetection

import java.util.*

// import java.util.*

data class Patient (val id: UUID = UUID.randomUUID(),
                    var firstName: String = "",
                    var lastName: String = "",
                    var roomNumber: Int = 1,
                    var age: Int = 80,
                    var sessionHistory: MutableList<Calendar> = mutableListOf()
)